README.md

```diff
@@ -99,6 +99,37 @@ function MyComponent( props ) {
 | `fallback` | `ReactElement`    | `null`   | Element that will be rendered if the user is no admin          |
 | `children` | `ReactElement(s)` | `'null'` | Child components that will be rendered if the user is an Admin |
 
+
+## CustomBlockInserter
+This component is passed to an `InnerBlocks` instance to as it's `renderAppender` to provide a customized button that opens the Block Inserter.
+
+### Usage
+```js
+import { CustomBlockAppender } from '@10up/block-components';
+const MyComponent = ({clientId}) => {
+	<InnerBlocks
+		renderAppender={() => (
+			<CustomBlockAppender
+				className="accordion-item-appender"
```

Comment:
Should we namescope our classes so they don't interfere with anything?

Reply:
@fabiankaegy that's probably a good idea